### ISSUE TITLE
Cannot run nn.Recurrent(8,nn.LookupTable(10000,10),nn.Linear(10,10),nn.Sigmoid(),5) recently

### ISSUE THREAD

**Issue by Andybert** [torch/rnn#38]
Before I reinstall torch and rnn module, everything is OK. But recently I reinstalled torch and rnn module for some reason, then I found that the following error would raise when I run code
require 'rnn'
r=nn.Recurrent(8,nn.LookupTable(10000,10),nn.Linear(10,10),nn.Sigmoid(),5)

error message:
[string "r = nn.Recurrent(8,nn.LookupTable(10000,10),n..."]:1: attempt to call field 'Recurrent' (a nil value)
stack traceback:
        [string "r = nn.Recurrent(8,nn.LookupTable(10000,10),n..."]:1: in main chunk
        [C]: in function 'xpcall'
        /home/xxx/torch/install/share/lua/5.1/trepl/init.lua:679: in function 'repl'
        ...ngli/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:204: in main chunk
        [C]: at 0x00405d50

anybody can tell me how to fix it?

**Comment by amartya18x** [comment]
I believe `nn.Recurrent` is not supported here anymore. You should be able to achieve similar results by using the `nn.Recurrence` module though I think. Check this out. `https://github.com/torch/rnn/blob/master/doc/recurrent.md#recurrence`
